docs/workbench/app-builder/add-a-map.mdx

```diff
@@ -5,13 +5,13 @@ tags: [app]
 sidebar_position: 2
 ---
 
-For many geospatial applications you will want to add a map to your Fused App, especially if your UDF returns a [Map `Tile`](/core-concepts/filetile/#map-tiles). 
+For many geospatial applications you will want to add a map to your Fused App, especially if your UDF returns a [Map `Tile`](/core-concepts/filetile/#map-tiles).
 
 This section shows a few examples of how you can do that. While we do recommend you use [`pydeck`](/workbench/app-builder/app-map/#pydeck) (the Python implementation of [deck.gl](https://deck.gl/)) for its versatility, you can use other options like [`folium`](/workbench/app-builder/app-map/#folium)
 
 :::note
 
-    Note that you need to install dependencies with `micropip` inside your Fused app. More on [this here](/workbench/app-builder/app-overview/#dependencies)
+    Note that you need to install dependencies with `micropip` inside your Fused app. More on this [here](/workbench/app-builder/app-overview/#dependencies).
```

Comment:
This is the only purposeful change. All others are extra space deletions done by the linter.